internal/checks/syncing.go

```
@@ -109,6 +108,8 @@ func (c *SyncingCheck) runCheck() {
 }
 
 func (c *SyncingCheck) IsPassing() bool {
+	// TODO(polsar): This method is unused. Instead, the decision whether this check is passing is made here:
+	//  https://github.com/satsuma-xyz/node-gateway/blob/b7f20aa2ad97f53772e9fa1565a300be7c0fff78/internal/route/node_filter.go#L98
```

same with this one, remove the implementation and fail hard.